crates/bevy_render/src/camera/projection.rs

```diff
@@ -161,49 +169,48 @@ pub enum ScalingMode {
+/// the size at which objects appear remain the same regardless of depth.
 ///
-/// If the window is smaller than the view frustum, some objects within the view frustum will not appear on the screen.
+/// The volume contained in the projection is called the *view frustum*. Since the viewport is rectangular,
+/// the view frustum is in the shape of a rectangular prism.
 ///
 /// Note that the cross sectional area of the view frustum and the apparent size of objects are inversely proportional.
 #[derive(Component, Debug, Clone, Reflect, FromReflect)]
 #[reflect(Component, Default)]
 pub struct OrthographicProjection {
-    /// The location of the left face of the camera's view frustum relative to the `window_origin`, measured in
-    /// world units.
+    /// The distance of the left face of the view frustum relative to the camera, measured in world units.
     ///
     /// This value will be set automatically unless `scaling_mode` is set to [`ScalingMode::None`].
     pub left: f32,
-    /// The location of the right face of the camera's view frustum relative to the `window_origin`, measured in
-    /// world units.
+    /// The distance of the right face of the view frustum relative to the camera, measured in world units.
     ///
     /// This value will be set automatically unless `scaling_mode` is set to [`ScalingMode::None`].
     pub right: f32,
-    /// The location of the bottom face of the camera's view frustum relative to the `window_origin`, measured in
-    /// world units.
+    /// The distance the bottom face of the view frustum relative to the camera, measured in world units.
     ///
     /// This value will be set automatically unless `scaling_mode` is set to [`ScalingMode::None`].
     pub bottom: f32,
-    /// The location of the top face of the camera's view frustum relative to the `window_origin`, measured in
-    /// world units.
+    /// The distance of the top face of the view frustum relative to the camera, measured in world units.
     ///
     /// This value will be set automatically unless `scaling_mode` is set to [`ScalingMode::None`].
     pub top: f32,
-    /// The distance of the near clipping plane, in world units.
+    /// The distance of the near clipping plane in world units.
     ///
     /// Objects closer than this will not be rendered.
     pub near: f32,
-    /// The distance of the far clipping plane, in world units.
+    /// The distance of the far clipping plane in world units.
     ///
     /// Objects further than this will not be rendered.
     pub far: f32,
-    /// Specifies where `(0, 0)` is located.
+    /// Specifies the origin of the scaling.
+    ///
+    /// This will not have any effect if `scaling_mode` is set to `ScalingMode::None`.
     pub window_origin: WindowOrigin,
     pub scaling_mode: ScalingMode,
-    /// Scales the cross sectional area of the view frustum.
+    /// Scales the width and height of the view frustum.
```

Comment:
This should be a bit more specific as the cross sectional area is a function of width and height, not the other way around.